react/features/toolbox/reducer.js

```diff
@@ -62,6 +61,20 @@ function _getInitialState() {
         hovered: false,
 
         /**
+         * Map containing default buttons for primary toolbar.
+         *
+         * @type {Map}
+         */
+        primaryToolbarButtons: new Map(),
+
+        /**
+         * Map containing default buttons for secondary toolbar.
+         *
+         * @type {Map}
+         */
+        secondaryToolbarButtons: new Map(),
```

Comment:
So you're introducing 2 new pieces of redux state in this commit and you're not reading it anywhere within the commit? It gives me the feeling that this commit fixes the problem that's the subject of this PR as a side effect.